src/midiEditorCommands.cpp

```diff
@@ -500,4 +500,23 @@ void cmdFocusNearestMidiEvent(Command* command) {
 		}
 	}
 }
+
+static BOOL CALLBACK cbFindFilterWindow(HWND hWnd, LPARAM lParam) {
+	WCHAR szText[128];
+	if ((GetWindowTextW(hWnd, szText, 128) != 0) &&
+		!wcsncmp(szText, L"Filter Events", 13)) {
+		*((HWND *)lParam) = hWnd;
+		return false;
+	}
+	return true;
+}
+
+void cmdMidiFilterWindow(Command *command) {
+	HWND editor = MIDIEditor_GetActive(), filter = NULL;
+	MIDIEditor_OnCommand(editor, command->gaccel.accel.cmd);
+	// TODO: we could also check the command state was "off", to skip searching otherwise
+	EnumWindows(cbFindFilterWindow, (LPARAM)&filter);
+	if (filter && (filter != GetFocus()))
+		SetFocus(filter); // focus the window
+}
```

Comment:
nit: Blank line before this, since it now encloses multiple functions.

Reply:
That I have not understood...

Reply:
Err, my comment was actually on the wrong line. Sorry. I meant please add a blank line before the #endif.

Reply:
I made the change myself. :)